src/components/AdvancedSearchOptions/AdvancedSearchOptionsPresentation.js

```diff
@@ -21,7 +21,12 @@ import {
 } from 'common/constants'
 
 function AdvancedSearchOptionsPresentation({ items }) {
-  const results = items?.pages[0]?.count
+  const count = items?.pages[0]?.count
+  let countStr = count
```

Comment:
Added separate variables here and in other files to not make changes to the original parameter and keep data types separate as well, e.g. count is a number, countStr denotes the string to be displayed.